new_course/1_programming_fundementals/debugging.md

```diff
@@ -70,11 +64,11 @@ end
 
 double_words_in_phrase("This is a test")
 
-["This", "is", "a", "test"]
+#=> ["This", "is", "a", "test"]
```

Comment:
I think we should differentiate what gets printed to STDOUT and what gets returned instead of using `#=>` for both cases.

Reply:
The reason that I added those in is that we also need to be able to differentiate between the code that's entered and what is getting printed to the console. 

What do you think of something like this?
~~~ruby
irb(main):001:0> def double_words_in_phrase(string)
irb(main):002:1>   string_array = string.split(' ')
irb(main):003:1>   string_array.map { |word| word * 2}
irb(main):004:1>   p string_array
irb(main):005:1>   string_array.join(' ')
irb(main):006:1> end
=> :double_words_in_phrase
irb(main):007:0>  double_words_in_phrase("This is a test")
["This", "is", "a", "test"]
=> "This is a test"
~~~ruby

Reply:
This would be the most correct, but it seems like a lot of work. How are we doing it elsewhere? I've forgotten.

Reply:
@leosoaivan I went ahead and used the irb syntax in this lesson for more clarity.